tools/openshift-ci/Dockerfile

```diff
@@ -2,13 +2,16 @@ FROM quay.io/centos/centos:8
 
 LABEL vendor="Red Hat inc."
 LABEL maintainer="OCP QE Team"
-
 USER root
 
-RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-Linux-* && \
-    sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-Linux-*
+ADD . /verification-tests/
```

Comment:
Since we do squash when building images in OpenShift, so single command / multiple commands do get the same image.

But multiple commands save up time/resource when building/testing image locally as we can reuse those existing layers and/or continue from the failed steps when do rebuild.

BTW, `ADD . /verification-tests/` will always have changes which will result image rebuild always starts as a fresh. So is possible we'd like to move it a few lines down to where it's needed.

Reply:
fixing